schematools/types.py

```diff
@@ -157,6 +215,10 @@ def relation(self) -> typing.Optional[str]:
     def format(self) -> typing.Optional[str]:
         return self.get("format")
 
+    @property
+    def is_nested_table(self) -> bool:
+        return field_is_nested_table(self)
```

Comment:
zou `field_is_nested_table()` hier niet inline kunnen staan?

Reply:
Deze functie was in DSO-API gebruikt, maar niet meer.